code/src/index.css

```diff
@@ -11,3 +12,147 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* Loading Screen */
+/* Overlay */
+.loading-overlay {
+  position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+  background: white;
+  display: flex;
+	justify-content: center;
+	align-items: center;
+  height: 500px;
+  width: 100%;
+}
+
+/* Bouncing loading ball */
+.loading-ball {
+  align-self: flex-end;
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+  background-color: #F44336;
+  height: 200px;
+  margin: 0 auto 0 auto;
+  border-radius: 50%;
+  transform-origin: bottom;
+  width: 200px;
+}
+
+.bounce {
+  animation-name: bounce;
+  animation-timing-function: ease;
+}
+
+@keyframes bounce {
+  0%   { transform: scale(1,1)      translateY(0); }
+  10%  { transform: scale(1.1,.9)   translateY(0); }
+  30%  { transform: scale(.9,1.1)   translateY(-100px); }
+  50%  { transform: scale(1.05,.95) translateY(0); }
+  57%  { transform: scale(1,1)      translateY(-7px); }
+  64%  { transform: scale(1,1)      translateY(0); }
+  100% { transform: scale(1,1)      translateY(0); }
+}
+
+/* All post containers  */
+.post-container {
+  width: 80%;
+  height: 175px;
+  margin: 20px auto;
+  padding: 10px 30px;
+  border: 2px solid black;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 5px 5px black;
+  position: relative;
+  word-break: break-word;
+}
+
+/* New post form */
+.post-container h1 {
+  font-size: 14px;
+}
+
+.form {
+  background-color: #f2f0f0;
+  position: relative;
+}
+
+/* Input */
+#newThought {
```

Comment:
even though using Ids is not wrong at all however it is consider coding convention that classes should be used for styling and Ids should left for Javascript